seleniumByTechnocredits/src/seleniumPractise/vrushali/AlertOK.java

```diff
@@ -0,0 +1,49 @@
+// Q2 Alert > Ok > Validate the string of Alert with hardcoded value.
+
+package seleniumPractise.vrushali;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import util.Init;
+
+public class AlertOK {
+
+	public static void alertOkValidation(WebDriver driver) throws InterruptedException {
+		driver.get("http://automationbykrishna.com/#");
+
+		driver.findElement(By.xpath("//a[@id= 'basicelements']")).click();
+		System.out.println("Basic element tab clicked");
+			
+		WebDriverWait wait = new WebDriverWait(driver, 30);
```

Comment:
Why are you created WebDriverWait object here, and don't use in this method?

Reply:
@krishnakTechnocretids I have created a WebDriverWait object because I wanted to use explicit wait in the program. The WebDriverWait object is used in line number 25.